sql/lenkwerk/storage/storage.sql

```diff
@@ -61,8 +61,8 @@ can be set using the default_val field though this is not yet supported.$$;
 create table storage.indexes (
    id serial not null unique, 
    indexname varchar(16) default 'bagger_idx',
-   access_method varchar 
-           check (access_method in ('gin', 'gist', 'btree', 'hash')),
```

Comment:
The constraint was removed so that we can support dynamic options for index access methods.  This is still enforced on the Perl side.